docs/tutorial/maps.rst

```diff
@@ -181,748 +166,386 @@ Some basic statistical functions are built into Map objects:
 
-.. code-block:: python
+.. _cropping-maps:
 
-    >>> plt.colorbar(extend='both')   # doctest: +SKIP
+Cropping Maps and Combining Pixels
+==================================
 
-Here is an example of this put to use on an AIA image.
+It analyzing images of the Sun, we often want to choose a smaller portion of the full disk to look at more closely.
+Let's use the region of interest we defined above to crop out that portion of our image.
 
 .. plot::
     :include-source:
+    :context: close-figs
 
-    import astropy.units as u
-    import matplotlib.pyplot as plt
-
-    import sunpy.map
-    import sunpy.data.sample
-
-    smap = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    cmap = smap.cmap.copy()
-    cmap.set_over('blue')
-    cmap.set_under('green')
-
-    fig = plt.figure(figsize=(12, 4))
+    my_submap = my_map.submap(roi_bottom_left, top_right=roi_top_right)
+    fig = plt.figure()
+    ax = fig.add_subplot(projection=my_submap)
+    my_submap.plot(axes=ax)
+    plt.show()
 
-    ax1 = fig.add_subplot(1, 2, 1, projection=smap)
-    smap.plot(title='Without clipping')
-    plt.colorbar()
+Additionally, we also may want to combine multiple pixels into a single pixel (a so called "superpixel") to, for example, increase our signal-to-noise ratio.
+We can accomplish this with the `~sunpy.map.GenericMap.superpixel` method by specifying how many pixels, in each dimension, we want our new superpixels to contain.
+For example, we can combine 4 pixels in each dimension such that our new superpixels contain 16 original pixels.
 
-    ax2 = fig.add_subplot(1, 2, 2, projection=smap)
-    smap.plot(clip_interval=(1, 99.5)*u.percent, title='With clipping')
-    plt.colorbar(extend='both')
+.. plot::
+    :include-source:
+    :context: close-figs
 
+    my_super_submap = my_submap.superpixel((5,5)*u.pixel)
+    fig = plt.figure()
+    ax = fig.add_subplot(projection=my_super_submap)
+    my_super_submap.plot(axes=ax)
     plt.show()
 
+.. note::
 
-Masking is another approach to ignoring certain data.
-A mask is a boolean array that can give you fine-grained control over what is not being displayed.
-The `~numpy.ma.MaskedArray` is a subclass of a NumPy array with the addition of an associated boolean array which holds the mask.
-See the following two examples for applications of this technique:
-
-* :ref:`sphx_glr_generated_gallery_computer_vision_techniques_mask_disk.py`
-
-* :ref:`sphx_glr_generated_gallery_computer_vision_techniques_finding_masking_bright_pixels.py`
+    Map provides additional methods for manipulating the underlying image data.
+    See the reference documentation for `~sunpy.map.GenericMap` for a complete list of available methods as well as the :ref:`Map section of the Example Gallery <sphx_glr_generated_gallery_map>` for more detailed examples.
 
 .. _map-sequences:
 
 Map Sequences
```

Comment:
This whole section seems a little light on details and motivation, and I wouldn't be against getting rid of it and starting a new map sequences/compositemap tutorial in future?

Reply:
Yeah I agree. I removed the CompositeMap section for this reason because I didn't think it added much information. I'm inclined to keep this at least for now only because without it, we essentially have no docs (besides reference) on using MapSequence.